service/src/main/resources/db/changesets/20230222_states.yaml

```diff
@@ -0,0 +1,43 @@
+databaseChangeLog:
+- changeSet:
+    id: Add state to resources and cloud contexts
+    author: dd
+    changes:
+    - addColumn:
+        tableName: resource
+        columns:
+            - column:
+                  name: state
+                  type: text
+                  remarks: |
+                      String for of the state enum. See WsmResourceState.
+            - column:
+                  name: flight_id
+                  type: text
+                  remarks: |
+                      For transition states (e.g., updating), the flight doing the transition
+            - column:
+                  name: error
+                  type: text
+                  remarks: |
+                      Serialized exception; must be a subclass of ErrorReportException
+    -addColumn:
+        tableName: cloud_context
```

Comment:
I don't see where this is used in this PR (outside of tests). if you're adding that in another PR can this change be part of that instead?

Reply:
OK